### DISCUSSION TITLE
langage programmation x-robot  pour les micocontroleurs stm32

### DISCUSSION THREAD

**Discussion by anicet86** [community/community#124024] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Bonjour a toute la communaute moi c est ANICET je suis etutiant nouveau dans la programmation.
j'ai un projet de fin d etude qui consiste a developper un controleur pour les objets mobils dynamique en utilisant les stm32. et le langage de programmation est les X-Robot qui est intergre dans l'IDE MARS .
le probleme c est que cet IDE ne permet pas encore de programmer les stm32  d'ou l'objet de mon theme de recheche .
comment ajouter un interpreteur a L'IDE enfin qu'il puissse permettre de programmer les stm32

**Comment by ebndev** [comment]
Hi @anicet86, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
